.github/workflows/sdks.yaml

```
@@ -0,0 +1,20 @@
+name: SDKs
+
+on: [push]
+
+jobs:
+  python:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["pypy3.10", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

As GitHub Action is not limitless and supporting multiple Python versions will be cumbersome, let's support only the latest version. 
```suggestion
        python-version: ["3.13"]
```